### PR TITLE
Add package description file

### PR DESCRIPTION
**Pull request by etiennebacher**
https://github.com/easystats/easystats/issues/314

### CODE REVIEW

File: R/performance-package.R [hunk 1]
```diff
@@ -22,7 +22,6 @@
 #' References: Lüdecke et al. (2021) \doi{10.21105/joss.03139}
 #'
 #' @docType package
-#' @aliases performance performance-package
 #' @name performance-package
 #' @keywords internal
 "_PACKAGE"
```
Comment (IndrajeetPatil) [inline R/performance-package.R]:
```suggestion
"_PACKAGE"

```

File: R/performance-package.R [hunk 2]
```diff
@@ -0,0 +1,27 @@
+#' \code{performance-package}
```
Comment (IndrajeetPatil) [inline R/performance-package.R:1]: Does this create problems?

```suggestion
#' \code{performance}
```

File: R/performance-package.R (hunk 1 above)
Comment (etiennebacher) [reply R/performance-package.R]: I already added a new line locally but apparently git doesn't show it. I also can't commit your suggestion so I suppose it's not feasible?

File: R/performance-package.R (hunk 2 above)
Comment (etiennebacher) [reply R/performance-package.R:1]: yes it creates a warning in R CMD check:
> Rd files with duplicated alias 'performance':
>     'model_performance.Rd' 'performance-package.Rd'

**Review by IndrajeetPatil — APPROVED** [review]
Thanks!

**Pull request merged**